src/earl-loader.js

```diff
@@ -0,0 +1,25 @@
+const fs = require('fs')
+const globby = require('globby')
+
+/**
+ * Load jsonld/earl report(s)
+ * - if there are multiple reports, this function concatanates them into one report
+ *
+ * @method earlLoader
+ * @param {String|String[]} pathOrGlob file path to jsonld/earl report
+ * @returns {Object[]}
+ */
+const earlLoader = async pathOrGlob => {
```

Comment:
I have a strong preference for using `function earlLoader () {}` here. The advantage of that is that the function name is going to show up in the stack trace if this ever throws an error or something. I know lots of people disagree with me on this, but I think making sure that named functions have that name show up in the stack trace makes life better, so we should always do that.